WardenApi/src/main/java/com/salesforce/dva/warden/dto/Policy.java

```diff
@@ -1,5 +1,164 @@
 package com.salesforce.dva.warden.dto;
 
-public class Policy {
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/**
+ * Policy Dto.
+ *
+ * @author  Jigna Bhatt (jbhatt@salesforce.com)
+ */
+@SuppressWarnings("serial")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Policy extends com.salesforce.dva.warden.dto.Entity {
+	//~ Instance fields ******************************************************************************************************************************
+
+	    private String servcie;		
+	    private String name;
+	    private List<String> owner  = new ArrayList<String>();
+	    private List<String> user  = new ArrayList<String>();
+	    private String subSystem;
+	    private String metricName;
+	    private String triggerType;
+	    private String aggregator;
+	    private List<Double> threshold;	    
+	    private String timeUnit;
+	    private double defaultValue;
+	    private String cronEntry;
+	    private List<BigInteger> suspensionLevels = new ArrayList<BigInteger>();
```

Comment:
It looks like the other DTO's actually do use the 'id' naming convention.  Apologies, but please change this back to 'suspensionLevelIds'